test/luatest_helpers/misc.lua

```diff
@@ -21,8 +26,15 @@ local function skip_if_enterprise()
     luatest.skip_if(is_enterprise_package(), 'package is Enterprise')
 end
 
+-- Skips a running test if Tarantool build target is macOS.
+local function skip_if_macOS()
+    luatest.skip_if(is_macOS_build(), 'build target is not Linux')
+end
+
 return {
     is_debug_build = is_debug_build,
+    is_macOS_build = is_macOS_build,
     skip_if_not_debug = skip_if_not_debug,
     skip_if_enterprise = skip_if_enterprise,
+    skip_if_macOS = skip_if_macOS,
```

Comment:
This file is about to be removed. The helpers should be added to the luatest repository, see https://github.com/tarantool/luatest/pull/247.

Reply:
Removed this helper in favor of testing for `readelf` availability.